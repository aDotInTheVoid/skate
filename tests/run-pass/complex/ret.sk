fn a() {
    b()
}

fn b() {
    c()
}

fn c() {
    d()
}

fn d() {
    "hello"
}

fn main() {
    print a();
    print b();
    print c();
    print d();
}