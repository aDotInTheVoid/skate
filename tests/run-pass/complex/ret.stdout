hello
hello
hello
hello
hellohellohellohello
